### PR TITLE
Directory separator for Linux and Windows

### PR DESCRIPTION
**Pull request by ccronca**

### CODE REVIEW

File: templates/input.erb [hunk 1]
```diff
@@ -10,6 +10,6 @@
   File        <%= "'#{@input_file_path}'" %>
   <%- end %>
   <%- if @input_type -%>
```
Comment (genebean) [inline templates/input.erb:12]: Why do you want to remove the single quotes from the resulting config file? Just trying to make sure I understand all the changes.

Comment (ccronca) [reply templates/input.erb:12]: InputType directive doesn't use single quotes. Nxlog fails to start if single quotes are present.
https://nxlog.org/node/295#config_inputtype

**Comment by genebean** [comment]
For the first time in a while I have some time to look at modules... I will try and get this reviewed and merged shortly.

**Comment by ccronca** [comment]
Any updates ?

**Comment by genebean** [comment]
@thespain and I are working on this again and I plan to try and finally address this and other open PR's. Sincerely apologize for failing at being responsive. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by genebean** [comment]
Fixed up locally and merged into `develop`. Thanks for the contribution oh so long ago.

**Pull request closed without merging**